src/molecules/ProfileDropdown.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import GenericDropdown from '../components/GenericDropdown';
+import styled from 'styled-components';
+
+const DropdownItems = [
+	{
+		title: 'Profile',
+		path: '/profile',
+		icon: 'fas fa-user-circle',
+	},
+	{
+		title: 'Settings',
+		path: '/settings',
+		icon: 'fas fa-cog',
+	},
+	{
+		title: 'Sign Out',
+		path: '/signout',
+		icon: 'fas fa-sign-out-alt',
+	},
+];
+
+const Button = styled.div`
+	padding: 1em;
+`;
+
+function ProfileDropdown() {
+	return (
+		<GenericDropdown
+			buttonDisplay={
+				<Button>
+					{'"Name of User"  '}
```

Comment:
Maybe, depending on what the dropdown's purpose is, the default value could be `Log in` or something? (Unless the user logs in before they actually see the dropdown; that's a bigger thing to think about)

Reply:
For this one I just tried to imitate the Figma  mock up (https://www.figma.com/file/YD5BwYLLDdoNx6u2sanuyn/Passr-style-guide?node-id=7985%3A94), and then we can replace the `{'"Name of User'"}` with actual names. I assumed that the ProfileDropdown would only be shown after logging in since the dropdown items consist of 'Profile', 'Settings', and 'Sign out'.

Reply:
Yeah, I will add to this and say the profile dropdown is only present if the user is logged in. Otherwise we will present an action button to log in.